skunkworks_crow/src/main/java/org/odk/share/activities/SendActivity.java

```diff
@@ -130,6 +130,12 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     public void onBackPressed() {
+        if (!isHotspotRunning) {
+            stopHotspot();
```

Comment:
If the hotspot is not enabled then why are we calling `stopHotspot`?

Reply:
I blindly copy pasted positive button callback from `stopHotspotAlertDialog`
I agree and understand that block is unnecessary now.